frontend/js/behaviors/core/toggleFee.js

```diff
@@ -0,0 +1,35 @@
+const toggleFee = function(container) {
+
+    container.addEventListener('click', function() {
+        showFee.call(this);
+            selectCategory.call(this);
```

Comment:
Run `composer lint` on your code to clean up some of these formatting discrepencies.

Reply:
Since this is Javascript, I don't think our linters will pick it up. We could use something like https://prettier.io/ for linting and formatting our JS.